Index: src/main/resources/com/example/demo1/home-view.fxml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\n\n<?import javafx.scene.layout.AnchorPane?>\n<?import javafx.scene.layout.VBox?>\n\n\n<AnchorPane prefHeight=\"400.0\" prefWidth=\"600.0\" xmlns=\"http://javafx.com/javafx/21\" xmlns:fx=\"http://javafx.com/fxml/1\" fx:controller=\"com.example.demo1.Controller.HomeController\">\n   <children>\n      <VBox prefHeight=\"400.0\" prefWidth=\"100.0\" />\n      <VBox layoutX=\"500.0\" prefHeight=\"400.0\" prefWidth=\"100.0\" />\n   </children>\n</AnchorPane>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/com/example/demo1/home-view.fxml b/src/main/resources/com/example/demo1/home-view.fxml
--- a/src/main/resources/com/example/demo1/home-view.fxml	(revision a6929263c74fb504b583c7e197e3867a29d816a6)
+++ b/src/main/resources/com/example/demo1/home-view.fxml	(date 1714294666599)
@@ -4,7 +4,7 @@
 <?import javafx.scene.layout.VBox?>
 
 
-<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/21" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.demo1.Controller.HomeController">
+<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/21" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.example.demo1.HomeController.HomeController">
    <children>
       <VBox prefHeight="400.0" prefWidth="100.0" />
       <VBox layoutX="500.0" prefHeight="400.0" prefWidth="100.0" />
Index: src/main/java/com/example/demo1/Controller/HomeController.java
===================================================================
diff --git a/src/main/java/com/example/demo1/Controller/HomeController.java b/src/main/java/com/example/demo1/Controller/HomeController.java
deleted file mode 100644
--- a/src/main/java/com/example/demo1/Controller/HomeController.java	(revision a6929263c74fb504b583c7e197e3867a29d816a6)
+++ /dev/null	(revision a6929263c74fb504b583c7e197e3867a29d816a6)
@@ -1,4 +0,0 @@
-package com.example.demo1.Controller;
-
-public class HomeController {
-}
Index: src/main/java/com/example/demo1/HomeController/HomeController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/example/demo1/HomeController/HomeController.java b/src/main/java/com/example/demo1/HomeController/HomeController.java
new file mode 100644
--- /dev/null	(date 1714294666594)
+++ b/src/main/java/com/example/demo1/HomeController/HomeController.java	(date 1714294666594)
@@ -0,0 +1,4 @@
+package com.example.demo1.HomeController;
+
+public class HomeController {
+}
Index: src/main/java/module-info.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>module com.example.demo1 {\n    requires javafx.controls;\n    requires javafx.fxml;\n    requires java.datatransfer;\n    requires jbcrypt;\n    requires java.sql;\n\n\n    exports com.example.demo1;\n    exports com.example.demo1.Controller;\n\n    opens com.example.demo1 to javafx.fxml;\n    opens com.example.demo1.Controller to javafx.fxml;\n    exports com.example.demo1.Models;\n    opens com.example.demo1.Models to javafx.fxml;\n\n    exports com.example.demo1.Calendar;\n    opens com.example.demo1.Calendar to javafx.fxml;\n\n    exports com.example.demo1.AccountController;\n    opens com.example.demo1.AccountController to javafx.fxml;\n    exports com.example.demo1.AccountModel;\n    opens com.example.demo1.AccountModel to javafx.fxml;\n\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/module-info.java b/src/main/java/module-info.java
--- a/src/main/java/module-info.java	(revision a6929263c74fb504b583c7e197e3867a29d816a6)
+++ b/src/main/java/module-info.java	(date 1714294666584)
@@ -7,10 +7,10 @@
 
 
     exports com.example.demo1;
-    exports com.example.demo1.Controller;
+    exports com.example.demo1.HomeController;
 
     opens com.example.demo1 to javafx.fxml;
-    opens com.example.demo1.Controller to javafx.fxml;
+    opens com.example.demo1.HomeController to javafx.fxml;
     exports com.example.demo1.Models;
     opens com.example.demo1.Models to javafx.fxml;
 
