Index: src/main/java/com/example/demo1/HelloApplication.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.example.demo1;\n\nimport javafx.application.Application;\nimport javafx.fxml.FXMLLoader;\nimport javafx.scene.Scene;\nimport javafx.stage.Stage;\n\nimport java.io.IOException;\n\npublic class HelloApplication extends Application {\n    @Override\n    public void start(Stage stage) throws IOException {\n        FXMLLoader fxmlLoader = new FXMLLoader(HelloApplication.class.getResource(\"hello-view1.fxml\"));\n        Scene scene = new Scene(fxmlLoader.load(), 320, 240);\n        stage.setTitle(\"Profile Settings\");\n        stage.setMinWidth(800);\n        stage.setMinHeight(500);\n        stage.setScene(scene);\n        stage.show();\n    }\n\n    public static void main(String[] args) {\n        launch();\n    }\n}
===================================================================
diff --git a/src/main/java/com/example/demo1/HelloApplication.java b/src/main/java/com/example/demo1/HelloApplication.java
--- a/src/main/java/com/example/demo1/HelloApplication.java	
+++ b/src/main/java/com/example/demo1/HelloApplication.java	
@@ -10,7 +10,7 @@
 public class HelloApplication extends Application {
     @Override
     public void start(Stage stage) throws IOException {
-        FXMLLoader fxmlLoader = new FXMLLoader(HelloApplication.class.getResource("hello-view1.fxml"));
+        FXMLLoader fxmlLoader = new FXMLLoader(HelloApplication.class.getResource("profile-view.fxml"));
         Scene scene = new Scene(fxmlLoader.load(), 320, 240);
         stage.setTitle("Profile Settings");
         stage.setMinWidth(800);
@@ -22,4 +22,5 @@
     public static void main(String[] args) {
         launch();
     }
-}
\ No newline at end of file
+}
+
diff --git a/src/main/resources/com/example/demo1/hello-view1.fxml b/src/main/resources/com/example/demo1/profile-view.fxml
rename from src/main/resources/com/example/demo1/hello-view1.fxml
rename to src/main/resources/com/example/demo1/profile-view.fxml
